package/yaft/package

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(yaft)
+pkgdesc="Framebuffer terminal emulator with an on-screen touch keyboard"
+url=https://github.com/timower/rM2-stuff/tree/master/apps/yaft
+pkgver=0.0.1-1
+timestamp=2020-10-27T12:02Z
+section="admin"
+maintainer="Timothy Werquin"
```

Comment:
If you become the maintainer of this new package, you’ll be responsible for keeping it up to date and test new releases. (See https://github.com/toltec-dev/toltec/blob/stable/docs/branches.md.) If you don’t want to take on this duty, one of the repository maintainers can take ownership of the package. Otherwise, please add a contact email in this field.

Reply:
Ok, If I'm not a maintainer should I set it to `None` for now?

Reply:
The question is, do you want to maintain this package or to leave it to one of us? If you don't want to, you'd set it to `None <none@example.org>`